src/encoding.iced

```diff
@@ -29,14 +29,15 @@ exports.Encoding = class Encoding
     encoded_bits - decoded_bits
 
   # encode a block of length @in_block_len or less....
-  encode_block : (block) ->
+  encode_block : (block, old_shift) ->
```

Comment:
Let's include some sort of doc-style comment above this function, explaining what the `old_shift` argument does. You can make reference to the "Comparison to Base64" section of the saltpack armor spec.